c7n/resources/swf.py

```diff
@@ -0,0 +1,87 @@
+# Copyright The Cloud Custodian Authors.
+# SPDX-License-Identifier: Apache-2.0
+from c7n.manager import resources
+from c7n.query import ConfigSource, DescribeSource, QueryResourceManager, TypeInfo
+from c7n.tags import Tag, RemoveTag, universal_augment
+
+
+class DescribeSimpleWorkflow(DescribeSource):
+    def augment(self, resources):
+        return universal_augment(self.manager, super().augment(resources))
+
+
+@resources.register('swf')
+class SimpleWorkflow(QueryResourceManager):
+    class resource_type(TypeInfo):
+        service = 'swf'
+        arn_type = ''
+        enum_spec = ('list_domains', 'domainInfos', {'registrationStatus': 'REGISTERED'})
+        id = name = 'name'
+        arn = 'arn'
+        universal_taggable = object()
+        permission_augment = ('swf:ListTagsForResource',)
+
+    source_mapping = {
+        'describe': DescribeSimpleWorkflow,
+        'config': ConfigSource
+    }
+
+
+@SimpleWorkflow.action_registry.register('tag')
```

Comment:
does this even need a tag action with universal_taggable?

Reply:
You're right, just removed that.

I initially tried `universal_taggable` but ran into this https://github.com/cloud-custodian/cloud-custodian/issues/6515 , so I went with the manual implementation. Had to update my local version of Custodian and all is well with `universal_taggable` now.